pkg/asset/machines/azure/azuremachines.go

```diff
@@ -133,6 +133,12 @@ func GenerateMachines(platform *azure.Platform, pool *types.MachinePool, userDat
 				AllocatePublicIP:       false,
 				EnableIPForwarding:     false,
 				SecurityProfile:        securityProfile,
+				NetworkInterfaces: []capz.NetworkInterface{
+					{
+						SubnetName:            subnet,
+						AcceleratedNetworking: additionalCapabilities.UltraSSDEnabled,
```

Comment:
I think this should just be set to true. SSD capabilities are for disks, not networking interfaces.